SharkCage/SharedFunctionality/SharedFunctions.cpp

```diff
@@ -134,6 +136,14 @@ namespace SharedFunctions
 				config_stream >> json_config;
 
 				auto path = json_config[APPLICATION_PATH_PROPERTY].get<std::string>();
+				
+				auto version = json_config[CONFIG_VERSION_PROPERTY].get<int>();
+				
+				std::string cmd_line = "";
+				if (version >= 2)
+				{
+					cmd_line = json_config[APPLICATION_CMD_LINE_PROPERTY].get<std::string>();
```

Comment:
`""` could be replaced with `String.Empty` or?

Reply:
not in C++, no 😅